networkit/cpp/generators/MocnikGenerator.cpp

```diff
@@ -292,11 +292,11 @@ Graph MocnikGenerator::generate() {
 	assert(dim > 0);
 	for (auto &n : ns) {
```

Comment:
Rewrite as `assert(std::all_of(ns.cbegin(), ns.cend(), [] (count n) {return n >1;}));` and `assert(std::all_of(ks.cbegin(), ks.cend(), [] (double k) {return k >1.0;}));`